spec/AuthenticationAdapters.spec.js

```diff
@@ -576,4 +576,26 @@ describe('google auth adapter', () => {
       expect(e.message).toBe('Google auth is invalid for this user.');
     }
   });
+
+  it('properly verify a game center identity', () => {
+    const authenticationHandler = authenticationLoader({
+      gcenter: path.resolve('./src/Adapters/Auth/gcenter.js')
+    });
+     validateAuthenticationHandler(authenticationHandler);
```

Comment:
`ReferenceError: validateAuthenticationHandler is not defined`

This is causing tests to fail...